[FEATURE] Allow non cachable plugins to return null on empty

twoldanski opened the pull request:
Handles cases when plugin marked as USER_INT wants to in some case do not return some data, by default instead of empty string, we want set null for better type system in FE app

tmotyl commented:
@twoldanski please give more description to the ticket (what usecase is solving)

twoldanski committed:
[FEATURE] Allow non cachable plugins to return null on empty
Handles cases when plugin marked as USER_INT wants to in some case do not return some data, by default instead of empty string, we want set null for better type system in FE app

lukaszuznanski committed:
Merge branch '3.x' into feat/headless-user-intv3

lukaszuznanski committed:
Merge branch '3.x' into feat/headless-user-intv3
